### DISCUSSION TITLE
Send multiple rows to a mssql query

### DISCUSSION THREAD

**Discussion by BRM-SMS** [Budibase/budibase#10260] [Help]
I need to update and insert data to a external mssql database. ive already written the query but i can only send 1 row to the query.
i also tried automation, but i couldnt make it.

how do i send multiple rows to be executes with the update/insert query?

![image](https://user-images.githubusercontent.com/119508535/230406714-500fc1cf-fc9f-4325-8cc2-aba8eee46499.png)

**Comment by melohagan** [comment]
Hey @BRM-SMS 

What is your use-case that you need to save multiple rows in one form? Could you use form steps instead, and save one row per step?

**Reply by BRM-SMS** [reply]
Use-case is to save multiple rows in a table on the budibase database, then send all rows to a external mssql database. i tried it with looping, but it seems broken and inserting one value multiple times.

**Reply by melohagan** [reply]
And why do you need to save rows into a Budibase DB and then send them all to an external MS SQL DB later. Can you not use MS SQL DB directly in your app?

**Reply by BRM-SMS** [reply]
> And why do you need to save rows into a Budibase DB and then send them all to an external MS SQL DB later. Can you not use MS SQL DB directly in your app?

I wanted to use a barcode scanner to create a temporary table, then approve the entries, then send it to a mssql db. Sending it directly to mssql in a production environment would be stupid. Also i gave up on Budibase and trying something else
